app/graph/types/project_media_type.rb

```diff
@@ -85,9 +85,14 @@
 
   field :annotations_count do
     type types.Int
+    argument :annotation_type, types.String
 
-    resolve ->(project_media, _args, _ctx) {
-      project_media.get_versions_log_count
+    resolve ->(project_media, args, _ctx) {
```

Comment:
No, and usually they won't be equal. For example, the `embed` annotation created by Pender, the first status that is created automatically, etc, are not counted in the `log_size`.